### DISCUSSION TITLE
LAN on WT32-ETH01

### DISCUSSION THREAD

**Discussion by SOliver78** [luc-github/ESP3D#1033] [General]
Hello,

i have build ESP3D v3.0.0-alpha3 for an WT32-ETH01 ESP32, but Networ Card dont enabled.

My configuration in configuration.h ist the following and Wifi Config ist deleted:
// Use Ethernet 
// Enable ethernet communications
#define ETH_FEATURE 

// Ethernet chip
// LAN8720
// Chip used by board
#define ESP3D_ETH_PHY_TYPE TYPE_ETH_PHY_LAN8720

// Ethernet board clock mode
// GPIO0 IN
// Mode used by clock
#define ESP3D_ETH_CLK_MODE MODE_ETH_CLOCK_GPIO0_IN

// Ethernet board address
// 1
// Address used by clock
#define ESP3D_ETH_PHY_ADDR 1

// Ethernet power pin
// The pin used to power ethernet chip
#define ESP3D_ETH_PHY_POWER_PIN 16

// Ethernet mdc pin
// The pin used for MDC signal
#define ESP3D_ETH_PHY_MDC_PIN 23

// Ethernet mdio pin
// The pin used for MDIO signal
#define ESP3D_ETH_PHY_MDIO_PIN 18

What is false?

Sorry for my bad english.

**Comment by luc-github** [comment]
What means :  Network Card dont enabled ? what s the problem ?

>ESP3D v3.0.0-alpha3

this one is very old please try the latest git 

>Sorry for my bad english.

Where you from ?

**Reply by SOliver78** [reply]
Hello,

I come from Germany.

The problem is that the WT32-ETH01 doesn't get a link. My network switch also says on the port that nothing is connected.

I'm now trying again with the version from the GIT.

**Comment by SOliver78** [comment]
With the version from the git it now works with the same configuration.h without any problems.

Thanks for the hint ;)
